support-workers/src/main/scala/com/gu/emailservices/FailedContributionEmailFields.scala

```diff
@@ -5,6 +5,9 @@ object FailedEmailFields {
   def contribution(email: String, identityUserId: IdentityUserId): EmailFields =
     failedEmailFields("contribution-failed", email, identityUserId)
 
+  def supporterPlus(email: String, identityUserId: IdentityUserId): EmailFields =
+    failedEmailFields("supporter-plus-failed", email, identityUserId) // TODO: this email doesn't exist!
```

Comment:
this is the email id in membership-workflow that is triggered when we end up in the support workers failure state (as opposed to a 24h timeout)

Reply:
Correct